model-governance/model_governance_graph.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Instead of linking to "here" link  to something descriptive like  "Graph notebook jupyter extension"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>